docs/src/index.md

```diff
@@ -1,8 +1,131 @@
+
+Please note that the package actually downloads a [precompiled binary](https://github.com/microsoft/LightGBM/releases)
+so you do not need to install LightGBM first. This is done as a user convenience, and support
+will be added for supplying ones own LightGBM binary (for GPU acceleration, etc).
+
+To add the package to Julia:
+```julia
+Pkg.add("LightGBM")
+```
+
+Running tests for the package requires the use of the LightGBM example files,
+download and extract the [LightGBM source](https://github.com/microsoft/LightGBM/archive/v2.3.1.zip)
+and set the enviroment variable `LIGHTGBM_EXAMPLES_PATH` to the root of the source installation.
+Then you can run the tests by simply doing
+```julia
+Pkg.test("LightGBM")
+```
+
+# A simple example using LightGBM example files
+
+First, download [LightGBM source](https://github.com/microsoft/LightGBM/archive/v2.3.1.zip) 
+and untar it somewhere.
+
+```bash
+cd ~
+wget https://github.com/microsoft/LightGBM/archive/v2.3.1.tar.gz
+tar -xf v2.3.1.tar.gz
+```
+
+```julia
+using LightGBM
+using DelimitedFiles
+
+LIGHTGBM_SOURCE = abspath("~/LightGBM-2.3.1")
+
+# Load LightGBM's binary classification example.
+binary_test = readdlm(joinpath(LIGHTGBM_SOURCE, "examples", "binary_classification", "binary.test"), '\t')
+binary_train = readdlm(joinpath(LIGHTGBM_SOURCE, "examples", "binary_classification", "binary.train"), '\t')
+X_train = binary_train[:, 2:end]
+y_train = binary_train[:, 1]
+X_test = binary_test[:, 2:end]
+y_test = binary_test[:, 1]
+
+# Create an estimator with the desired parameters—leave other parameters at the default values.
+estimator = LGBMClassification(
+    objective = "binary",
+    num_iterations = 100,
+    learning_rate = .1,
+    early_stopping_round = 5,
+    feature_fraction = .8,
+    bagging_fraction = .9,
+    bagging_freq = 1,
+    num_leaves = 1000,
+    num_class = 1,
+    metric = ["auc", "binary_logloss"]
+)
+
+# Fit the estimator on the training data and return its scores for the test data.
+fit!(estimator, X_train, y_train, (X_test, y_test))
+
+# Predict arbitrary data with the estimator.
+predict(estimator, X_train)
+
+# Cross-validate using a two-fold cross-validation iterable providing training indices.
+splits = (collect(1:3500), collect(3501:7000))
+cv(estimator, X_train, y_train, splits)
+
+# Exhaustive search on an iterable containing all combinations of learning_rate ∈ {.1, .2} and
+# bagging_fraction ∈ {.8, .9}
+params = [Dict(:learning_rate => learning_rate,
+               :bagging_fraction => bagging_fraction) for
+          learning_rate in (.1, .2),
+          bagging_fraction in (.8, .9)]
+search_cv(estimator, X_train, y_train, splits, params)
+
+# Save and load the fitted model.
+filename = pwd() * "/finished.model"
+savemodel(estimator, filename)
+loadmodel(estimator, filename)
 ```
 
-```@autodocs
-Modules = [LightGBM]
+# Exports
```

Comment:
You could probably rename this to Parameters and the same in the TOC

Reply:
Rename the section "Export" to "Parameters". Modify the table of contents in the same way